can_messages.asciipb

```diff
@@ -115,7 +115,7 @@ msg {
 msg {
   id: 8
   source: CHAOS
-  # target: PLUTUS, MOTOR_CONTROLLER, DRIVER_CONTROLS
+  # target: PLUTUS, MOTOR_CONTROLLER, DRIVER_CONTROLS_CENTER_CONSOLE
```

Comment:
Not sure how you're planning on ensuring devices are alive/data is not stale (or if that's even a concern you plan on dealing with), but if you're using heartbeats would you want to include all the other devices that form your "Distributed Driver Controls" in your heartbeat?

I imagine this is primarily a concern with respect to the Pedal, since you want to ensure that your master (which I'm assuming is the Center Console board) is still getting those values and can safely stop.

If so, please update the DBC script as well and make it aware that there will be ACK (as opposed to data) messages generated from the appropriate devices.